smithy-trait-codegen/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+description = "Plugin for Generating Trait Code from Smithy Models"
+
+ext {
+    set("displayName", "Smithy :: Trait Code Generation")
+    set("moduleName", "software.amazon.smithy.traitcodegen")
+}
+
+dependencies {
+    implementation(project(":smithy-codegen-core"))
+}
+
+// Set up Integration testing source sets
+sourceSets {
+    create("it") {
+        compileClasspath += sourceSets.main.get().output +
+                configurations["testRuntimeClasspath"] +
+                configurations["testCompileClasspath"]
+
+        runtimeClasspath += output +
+                compileClasspath +
+                sourceSets.test.get().runtimeClasspath +
```

Comment:
I would suggest just adding a block for: 
```
val main by getting
val test by getting
```
Then you can change ` sourceSets.test.get().runtimeClasspath` and such to `test.runtimeClasspath`